index.js

```diff
@@ -0,0 +1,77 @@
+import { displayBooksControl, removeBook, addBook } from './modules/display.js';
+import {
+  displayBooks, displayForm, contact, displayContact,
+  showBooksOnly, showOnlyContact, showOnlyForm,
+} from './modules/view.js';
+
+import { DateTime } from './modules/luxon.js';
+
+const bookItems = document.querySelector('#book-list');
+const awesomeHeading = document.querySelector('#awesome');
+
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+// display  all the books
+window.onload = displayBooksControl.displayBooks();
+
+// add books by inputting into form
+
+const form = document.querySelector('form');
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+
+  const title = document.querySelector('#title').value;
+  const author = document.querySelector('#author').value;
+  // validating form fields
+  if (title === '' || author === '') {
+    // eslint-disable-next-line no-alert
+    alert('You must fill in all fields');
+  } else {
+    const book = new Book(title, author);
+    awesomeHeading.style.display = 'block';
+    bookItems.style.display = 'flex';
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/194134125-26bb8882-1cdb-4a1e-a633-78a726c6a2ab.png)


- [ ] After adding a book it shows with the other books above the form it is not expected behaviour kindly fix it :+1: